### PR TITLE
Add functionality to retrieve deployed integrations by id

### PR DESCRIPTION
**Pull request by lootag**

### CODE REVIEW

File: go/examples/outgoing-webhooks_test.go [hunk 1]
```diff
@@ -92,17 +93,6 @@ func TestWebhooks(t *testing.T) {
 		},
 	})
 
-	crud(t, &cxsdk.CreateOutgoingWebhookRequest{
-		Data: &cxsdk.OutgoingWebhookInputData{
```
Comment (celaus) [inline go/examples/outgoing-webhooks_test.go:96]: Leave that one in until we replace it with the actual replacement (Microsoft Flow or whatever)

**Review by celaus — APPROVED** [review]

File: go/scripts/generate_go_protos.sh [hunk 2]
```diff
@@ -59,12 +59,13 @@ do
             *dashboards/v1/ast/widgets/common/queries.proto ) 
             ;; 
         *) 
-            protoc --proto_path=$proto_dir --go_out=../.. --go-grpc_out=../.. $args $proto_file
+            protoc --proto_path=$proto_dir --go_out=../.. --go-grpc_out=../.. --go-grpc_opt=module=github.com/coralogix --go_opt=module=github.com/coralogix $args $proto_file
```
Comment (celaus) [inline go/scripts/generate_go_protos.sh:62]: can you move the prefix to a variable and also use it as part of `mod_name`

File: go/examples/integrations_test.go [hunk 3]
```diff
@@ -21,6 +21,7 @@ import (
 	"testing"
 
 	cxsdk "github.com/coralogix/coralogix-management-sdk/go"
+	v1 "github.com/coralogix/coralogix-management-sdk/go/internal/coralogix/integrations/v1"
```
Comment (celaus) [inline go/examples/integrations_test.go:24]: never import anything from internal

File: go/examples/integrations_test.go [hunk 4]
```diff
@@ -111,23 +112,25 @@ func TestIntegration(t *testing.T) {
...
-	integrationDetail := details.IntegrationDetail.IntegrationTypeDetails.(*cxsdk.IntegrationDetailsDefault)
-	found := false
-	for _, d := range integrationDetail.Default.Registered {
-		if d.Id.Value == createResponse.IntegrationId.Value {
-			found = true
-			break
-		}
+	assert.Nil(t, listError)
+
+	deployedIntegration, getError := c.Get(context.Background(), &v1.GetDeployedIntegrationRequest{
```
Comment (celaus) [inline go/examples/integrations_test.go:124]: Re-export GetDeployedIntegrationRequest

File: rust/cx-sdk/src/client/integrations.rs [hunk 5]
```diff
@@ -167,7 +167,7 @@ impl IntegrationsClient {
     /// # Arguments
     /// * `id` - The id of the integration to retrieve.
     /// * `include_testing_revision` - Whether to include the testing revision.
-    pub async fn get(
+    pub async fn get_details(
```
Comment (celaus) [inline rust/cx-sdk/src/client/integrations.rs:170]: isn't this called list in the Go client

Comment (celaus) [reply rust/cx-sdk/src/client/integrations.rs:170]: Whatever the name it should be consistent

File: rust/examples/integrations/src/lib.rs [hunk 6]
```diff
@@ -98,21 +98,14 @@ mod tests {
             .await
             .unwrap();
 
-        let all_instances = client.get(name.clone(), true).await.unwrap();
```
Comment (celaus) [inline rust/examples/integrations/src/lib.rs:101]: Here too leave the old one in

**Review by celaus — APPROVED** [review]

File: go/examples/integrations_test.go [hunk 7]
```diff
@@ -111,23 +112,25 @@ func TestIntegration(t *testing.T) {
...
 	})
-	if retrievalError != nil {
-		log.Fatal(retrievalError.Error())
+	if listError != nil {
+		log.Fatal(listError.Error())
 	}
-	assert.Nil(t, retrievalError)
-	integrationDetail := details.IntegrationDetail.IntegrationTypeDetails.(*cxsdk.IntegrationDetailsDefault)
-	found := false
-	for _, d := range integrationDetail.Default.Registered {
```
[the remark this answers is not in the document]
Comment (lootag) [reply go/examples/integrations_test.go]: Done

File: go/examples/outgoing-webhooks_test.go (hunk 1 above)
Comment (lootag) [reply go/examples/outgoing-webhooks_test.go]: Done

File: go/scripts/generate_go_protos.sh (hunk 2 above)
Comment (lootag) [reply go/scripts/generate_go_protos.sh]: Done :-)

File: go/examples/integrations_test.go (hunk 3 above)
Comment (lootag) [reply go/examples/integrations_test.go]: Done

File: go/examples/integrations_test.go (hunk 4 above)
Comment (lootag) [reply go/examples/integrations_test.go]: Done

File: rust/cx-sdk/src/client/integrations.rs (hunk 5 above)
Comment (lootag) [reply rust/cx-sdk/src/client/integrations.rs:170]: Done

File: rust/examples/integrations/src/lib.rs (hunk 6 above)
Comment (lootag) [reply rust/examples/integrations/src/lib.rs:101]: Done

**Pull request merged**